lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -226,9 +229,10 @@ class ReportUIFeatures {
     this.headerSticky.style.transform = `translateY(${heightDiff * scrollPct * -1}px)`;
     this.headerBackground.style.transform = `translateY(${scrollPct * this.headerOverlap}px)`;
     this.lighthouseIcon.style.transform =
-      `translate3d(var(--report-width-half),` +
-      ` calc(-100% - ${scrollPct * this.headerOverlap * -1}px), 0) scale(${1 - scrollPct})`;
+      `translate3d(0,` +
+      ` calc(-${scrollPct * this.headerOverlap * -1}px), 0) scale(${1 - scrollPct})`;
     this.lighthouseIcon.style.opacity = (1 - scrollPct).toString();
+    this.headerMetadata.style.opacity = (1 - scrollPct).toString();
```

Comment:
since `.lh-header` only contains the icon and the metadata, maybe save `this.header` instead and set opacity once on the whole thing?

Reply:
Good call!